build.rs

```diff
@@ -19,6 +19,7 @@ fn generate_ts_bindings(ts_include_dir: String) -> Result<()> {
     println!("cargo:rerun-if-changed={}", header);
 
     let bindings = bindgen::Builder::default()
+        .clang_arg(format!("-I{}", ts_include_dir))
```

Comment:
What is this for?

Reply:
To access rpc_caller_session.h header file.

Reply:
This file provides the new RPC calls.

Reply:
This file provides the new RPC calls.

Reply:
This file provides the new RPC calls.